l1-contracts/foundry.toml

```diff
@@ -2,7 +2,7 @@
 allow_paths = ["../l2-contracts/contracts"]
 src = "contracts"
 out = "out"
-libs = ["node_modules", "../lib", "../da-contracts/"]
```

Comment:
ok, just pointing out this changed ( it was just renaming until now)

Reply:
yes, sorry, this was the only non-rename change